### PR TITLE
Adding transparent windows information to the Window API tutorial

### PR DESCRIPTION
**Pull request by akurasov**
Also closes https://github.com/JetBrains/compose-jb/issues/1339

### CODE REVIEW

File: tutorials/Window_API_new/README.md [hunk 1]
```diff
@@ -605,3 +605,46 @@ private fun WindowScope.AppWindowTitleBar() = WindowDraggableArea {
...
+                shape = RoundedCornerShape(20.dp) //window has round corners now
+            ) {
+                Text("Hello World!", color = Color.White)
+            }
+        }
+    }
+}
+```
+
+_Important note: Window transparency is implemented based on JDK implementation, that contains known issue in case of moving a Window between two monitors with different density. So when you move an App, the Window stops being transparent. And it seems nothing can be done with this situation on Compose side.
```
Comment (Rsedaikin) [inline tutorials/Window_API_new/README.md:649]: This is a Linux only issue. Everything is fine on macOS and Windows.

File: tutorials/Window_API_new/README.md [hunk 2]
```diff
@@ -605,3 +605,46 @@ private fun WindowScope.AppWindowTitleBar() = WindowDraggableArea {
 }
 ```
 <img alt="Draggable area" src="draggable_area.gif" height="239" />
+
+## Transparent windows (e.g. allows to make windows of a custom form)
+To create a transparent window it is enough to pass two parameners to the Window function: transparent=true and undecorate=true (it is not possible to decorate a transparent Window). Common scenario is to combine transparent window with a Surface of a custom form. Below is an example of a round-cornered Window.
```
Comment (Rsedaikin) [inline tutorials/Window_API_new/README.md:610]: I think we should quote parameters/methodы/classes and other appropriate names. Like
` transparent=true`

**Review by Rsedaikin — CHANGES_REQUESTED** [review]

File: tutorials/Window_API_new/README.md (hunk 1 above)
Comment (akurasov) [reply tutorials/Window_API_new/README.md]: fixed

File: tutorials/Window_API_new/README.md (hunk 2 above)
Comment (akurasov) [reply tutorials/Window_API_new/README.md]: fixed

**Review by Rsedaikin — APPROVED** [review]

**Pull request merged**